Exercises/SpaceAge/SpaceAge.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercises.SpaceAge
+{
+    internal class SpaceAge
+    {
+        private float _years;
+        private float _informedYear;
+
+        public SpaceAge(int seconds)
+        {
+            float yearInSeconds = 31557600f;
+            _informedYear = seconds / yearInSeconds;
+        }
+
+        public float OnEarth()
+        {
+            return _informedYear;
+        }
+
+        public float OnMercury()
+        {
+            return _informedYear / orbitalPeriodInYears("Mercury");
+        }
+
+        public float OnVenus()
+        {
+            return _informedYear / orbitalPeriodInYears("Venus");
+        }
+
+        public float OnMars()
+        {
+            return _informedYear / orbitalPeriodInYears("Mars");
+        }
+
+        public float OnJupiter()
+        {
+            return _informedYear / orbitalPeriodInYears("Jupiter");
+        }
+
+        public float OnSaturn()
+        {
+            return _informedYear / orbitalPeriodInYears("Saturn");
+        }
+
+        public float OnUranus()
```

Comment:
[OPCIONAL]: Considere o uso de arrow function (`=>`) quando for uma única linha de código.